Audit/Controllers/AuditController.cs

```diff
@@ -21,45 +21,80 @@ public class AuditController : ControllerBase
         public AuditController(IOptions<EnvironmentConfig> configuration, ILogger<AuditController> logger)
         {
             _logger = logger;
-            _repository = new Repository.AuditRepository(configuration,_logger);
+            _repository = new Repository.AuditRepository(configuration, _logger);
         }
 
         // GET: Audit
         [HttpGet]
         public async Task<JsonResult> Get()
         {
-            _logger.LogInformation("audit/get");
-            return new JsonResult(await _repository.FindAll().ConfigureAwait(true));
+            try
+            {
+                return new JsonResult(await _repository.FindAll().ConfigureAwait(true));
+            }
+            catch (Exception e)
+            {
+                return new JsonResult(e.Message);
+            }
         }
         // GET: Audit/id
         [HttpGet("{id}")]
         public async Task<JsonResult> GetbyId(string id)
         {
-            return new JsonResult(await _repository.FindById(id,true).ConfigureAwait(true));
+            try
+            {
+                return new JsonResult(await _repository.FindById(id, true).ConfigureAwait(true));
+            }
+            catch (Exception e)
+            {
+                return new JsonResult(e.Message);
+            }
         }
 
         // POST: Audit
         [HttpPost]
         public async Task<JsonResult> Post([FromBody] Entity.Audit value)
         {
-            return new JsonResult(await _repository.Create(value).ConfigureAwait(true));
+            try
+            {
+                return new JsonResult(await _repository.Create(value).ConfigureAwait(true));
+            }
+            catch (Exception e)
+            {
+                return new JsonResult(e.Message);
```

Comment:
We can return a logger message here, as on Delete(string id).

Reply:
Done